emop-maven-plugin/pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>emop</artifactId>
+    <groupId>edu.cornell</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>edu.cornell</groupId>
+  <artifactId>emop-maven-plugin</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>maven-plugin</packaging>
+
+  <name>emop-maven-plugin Maven Plugin</name>
+
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
```

Comment:
either remove or add the github URL

also, you should put shared attributes in the root pom.xml, e.g., so that they do not go out of synch